src/content/docs/tutorial-infrastructure-monitoring/investigate-spike-CPU.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Investigate a spike in CPU
+metaDescription: Solve system outages by investigating resource bottlenecks across your system infrastructure.
+---
+
+import selectApps from 'images/infrastructure_screenshot-full_hosts-app-tile.webp'
+
+import updatedSummary from 'images/infrastructure_screenshot-full_updated-summary.webp'
+
+import appsSummaryTable from 'images/infrastructure_screenshot-full_hosts-updated-summary-table.webp'
+
+You can use the infrastructure hosts pages to troubleshoot different incidents within your system. In the last doc, we discussed the benefits of fully instrumenting your system with recommendations from the guided install, then walked through a general example of how and why you might filter down to a set of hosts. This doc builds off that skill and walks you through how you might troubleshoot a real-world example.  
+
+## Objectives [#objectives]
+
+In this tutorial, you'll learn how to:
+
+* Find the cause of an unexpected load spike
+* Determine if resources are deprived because of a new deployment, or because of a job process
+
+## Investigate a sudden spike in load [#spike-load]
```

Comment:
we're moving back and forth between names here

 * page name: `Investigate a spike in CPU`
 * Objectives bullet: `an unexpected load spike`
 * next title: `a sudden spike in load`